### PR TITLE
add CVEProvidedMessage

### PR DESCRIPTION
**Pull request by bissenbay**
## Related Issues and Dependencies
https://github.com/thoth-station/cve-update-job/issues/373

### CODE REVIEW

**Comment by saisankargochhayat** [comment]
LGTM! :+1:

File: thoth/messaging/cve_provided.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+"""This is Thoth Messaging module for CVEProvidedMessage."""
+
+from typing import Optional
+
+from .message_base import MessageBase, BaseMessageContents
+
+
+class CVEProvidedMessage(MessageBase):
```
Comment (pacospace) [inline thoth/messaging/cve_provided.py:26]: This message should have new implementation I think, @KPostOffice WDYT?

Comment (pacospace) [reply thoth/messaging/cve_provided.py:26]: https://github.com/thoth-station/messaging/pull/300 this need to be merged before

Comment (bissenbay) [reply thoth/messaging/cve_provided.py:26]: WDYT @KPostOffice  ?

Comment (KPostOffice) [reply thoth/messaging/cve_provided.py:26]: @bissenbay Take a look at the messages in `messaging` now, the overall idea is still similar but requires a few changes

Comment (bissenbay) [reply thoth/messaging/cve_provided.py:26]: thanks, https://github.com/thoth-station/messaging/pull/306/commits/081bab6a92c28d9867ca753c155522ae8de507f1

**Review by KPostOffice — COMMENTED** [review]
Please add to __init__.py

**Comment by bissenbay** [comment]
> Please add to **init**.py

Thanks @KPostOffice , https://github.com/thoth-station/messaging/pull/306/commits/64ebe2fabcc50fd737eff51f25c21c4ba4a4f478

**Review by KPostOffice — COMMENTED** [review]
Please make this alphabetical. Also kebechet_run_url if you don't mind

**Review by KPostOffice — APPROVED** [review]

**Pull request merged**